package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "react-dates",
-  "version": "20.1.0",
+  "version": "20.1.1",
```

Comment:
similarly, please never update the version number in a PR; that only ever happens in a direct commit to master. (also, this would be a minor, not a patch, since it's adding something new)

Reply:
sure, removing this.